src/homepageExperience/containers/HomepageContainer.tsx

```diff
@@ -54,7 +54,15 @@ export const HomepageContainer: FC = () => {
   const squareGridCardSize = '200px'
 
   // events handling
-  const logPythonEvent = () => {
+  const logGoWizardClick = () => {
+    event('firstMile.goWizard.clicked')
+  }
+
+  const logNodeWizardClick = () => {
+    event('firstMile.goWizard.clicked')
```

Comment:
copypasta! (it says `goWizard` not `nodejsWizard`). Also, make sure it's `nodejsWizard` for consistency - all the other event names are `nodejsWizard`, rather than `nodeWizard`)

Reply:
Made function names consistent.